odata-data-aggregation-ext/3.6 Expressions Evaluable on a Collection.md

```diff
@@ -77,11 +77,11 @@ results in 2 (a third of 8, rounded down) entities. (This differs from `topperce
 ```
 :::
 
-A definition that is equivalent to a `$count` expression after a collection-valued path was made in [OData-URL, section 4.8](#ODataURL).
+A definition that is equivalent to a `$count` expression after a collection-valued path was made in [OData-URL, section "Addressing the Count of a Collection"]($$$OData-URL$$$#AddressingtheCountofaCollection).
 
 ## ##subsec Function `isdefined`
 
-Properties that are not explicitly mentioned in [`aggregate`](#Transformationaggregate) or [`groupby`](#Transformationgroupby) are considered to have been _aggregated away_. Since they are treated as having the null value in `$filter` expressions [OData-URL, section 5.1.1.15](#ODataURL), the `$filter` expression `Product eq null` cannot distinguish between an instance containing the value for the null product and the instance containing the aggregated value across all products (where the `Product` has been aggregated away).
+Properties that are not explicitly mentioned in [`aggregate`](#Transformationaggregate) or [`groupby`](#Transformationgroupby) are considered to have been _aggregated away_. Since they are treated as having the null value in `$filter` expressions [OData-URL,  section "Path Expressions"]($$$OData-URL$$$#PathExpressions), the `$filter` expression `Product eq null` cannot distinguish between an instance containing the value for the null product and the instance containing the aggregated value across all products (where the `Product` has been aggregated away).
```

Comment:
```suggestion
Properties that are not explicitly mentioned in [`aggregate`](#Transformationaggregate) or [`groupby`](#Transformationgroupby) are considered to have been _aggregated away_. Since they are treated as having the null value in `$filter` expressions [OData-URL, section "Path Expressions"]($$$OData-URL$$$#PathExpressions), the `$filter` expression `Product eq null` cannot distinguish between an instance containing the value for the null product and the instance containing the aggregated value across all products (where the `Product` has been aggregated away).
```